applications/management/commands/get_verified_addresses.py

```diff
@@ -0,0 +1,51 @@
+from django.core.management.base import BaseCommand, CommandError
+from applications import models
+import csv
+import easypost
+from time import sleep
+from django.conf import settings
+from app import settings as app_settings
+easypost.api_key = app_settings.EASYPOST_KEY
+
+
+class Command(BaseCommand):
+    help = 'Prints verified addresses for users'
+
+    def handle(self, *args, **options):
+        print('Gathering verified addresses...')
+
+        confirmed_applicants = models.Application.objects.filter(status='C')
+
+        with open(settings.EXPORT_FILES_URL + 'verified_apps.csv', 'w') as verified_csv:
+            csv_writer = csv.writer(verified_csv, delimiter=',', quotechar='"', quoting=csv.QUOTE_MINIMAL)
+            csv_writer.writerow(['Name', 'Email', 'Street', 'City', 'State', 'Zip', 'Country'])
+
+            for app in confirmed_applicants:
+                country = 'US'
+                if app.uniemail and '.ca' in app.uniemail:
+                    country = 'CA'
+                try:
+                    address = easypost.Address.create(
+                        verify=['delivery'],
+                        street1=app.address_line,
+                        street2=app.address_line_2,
+                        city=app.city,
+                        state=app.state,
+                        zip=app.zip_code,
+                        country=country
+                    )
+                    if address.verifications.delivery.success:
+                        street = app.address_line + (" " + app.address_line_2 if app.address_line_2 else '')
+                        res = [app.user.name, app.user.email, street, app.city, app.state, app.zip_code, "US"]
```

Comment:
Think the "US" part should be replaced by the country var when writing to the csv